### PR TITLE
Store minesweeper as a class

### PR DESCRIPTION
**Pull request by TheGovna**
Not completely sure if all of this is valid python. But redesigned the game in class form. Logic for left click is the same as before.

### CODE REVIEW

File: src/msaas/msaas.py [hunk 1]
```diff
@@ -27,57 +26,108 @@
...
+
+        self.num_mines_remaining -= 1
+        self.board[x][y].is_flagged = True
+
+    def is_mine(self, x, y):
+        return self.board[x][y].is_mine
+
+    def is_adjacent_to_mine(self, x, y):
+        return self.is_mine(x-1, y+1) or self.is_mine(x, y+1) or self.is_mine(x+1, y+1) or \
+            self.is_mine(x-1, y) or is_mine(x+1, y) or \
```
Comment (aarongolliver) [inline src/msaas/msaas.py]: fix formatting

File: src/msaas/msaas.py [hunk 2]
```diff
@@ -27,57 +26,108 @@
...
+            self.is_mine(x-1, y-1) or self.is_mine(x, y-1) or self.is_mine(x+1, y-1)
+
+class Cell:
+    def __init__(self):
+        self.is_clicked = False
+        self.is_flagged = False
+        self.is_mine = False
+
+############################################################
```
Comment (aarongolliver) [inline src/msaas/msaas.py]: Headers like this are a good hint that you should just make another file

File: src/msaas/msaas.py [hunk 3]
```diff
@@ -96,125 +146,43 @@ def get_game_state(game_id):
     else:
         raise Exception("Game not found!")
 
+############################################################
```
Comment (aarongolliver) [inline src/msaas/msaas.py]: same

File: src/msaas/msaas.py [hunk 4]
```diff
@@ -27,57 +26,108 @@
...
+            self.num_mines_remaining += 1
+            return
+
+        self.num_mines_remaining -= 1
+        self.board[x][y].is_flagged = True
+
+    def is_mine(self, x, y):
+        return self.board[x][y].is_mine
+
+    def is_adjacent_to_mine(self, x, y):
```
Comment (aarongolliver) [inline src/msaas/msaas.py]: is_adjacent_to_mine(0, 0) will always throw. As will anything else on the edges.

File: src/msaas/msaas.py (hunk 1 above)
Comment (TheGovna) [inline src/msaas/msaas.py]: what's wrong with the formatting?

**Pull request merged**